include/emqx_ct_job.hrl

```diff
@@ -14,4 +14,9 @@
 %% limitations under the License.
 %%--------------------------------------------------------------------
 
+-ifndef(EMQX_CT_JOB_HRL).
```

Comment:
Why the protection of the ifndef ?

Reply:
It’s to avoid error if included twice or more